src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,121 @@
+package duke;
+
+/**
+ * Parser executes all actions passed from the Ui class
+ */
+public class Parser {
+
+    TaskList tasks;
+
+    Parser(TaskList tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * prints the list of currently stored tasks
+     */
```

Comment:
As shown in the java coding standard guide, sentences in javadoc comments usually begin with a capital letter and end with a full-stop. It's definitely not a major error, but adhering closely to the coding standard could help achieve the overall goal of making your future tP code look like it was all written by a single person.